src/classBased/components/Header.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+const Header = () => {
+  const headerStyle = {
+    padding: '20px 0',
+    lineHeight: '1.5em',
+  };
+
+  return (
+    <header style={headerStyle}>
+      <h1
+        style={{
+          fontSize: '6rem',
+          fontWeight: '600',
+          marginBottom: '2rem',
+          lineHeight: '1em',
+          color: '#ececec',
+          textTransform: 'lowercase',
+          textAlign: 'center',
+        }}
+      >
+        todos
+      </h1>
+    </header>
```

Comment:
[OPTIONAL] 👍🏽 ;

Inline styles coded directly on tags would make your code look messy, but rather, you can use the React inline styles simply by adding the styles into a variable and passing it into the component/tag, in this case, your h1 tag, just as you did with the const headerStyle and passing it into the header tag <header style={headerStyle}>.

I think you can refactor your code to look something like this; 👇 

```
const header = {
         fontSize: '6rem',
          fontWeight: '600',
          marginBottom: '2rem',
          lineHeight: '1em',
          color: '#ececec',
          textTransform: 'lowercase',
          textAlign: 'center',
       }

return (
    <header style={headerStyle}>
        <h1 style ={header}>
            Todos
             ....
         </h1>
     </header>
  )
```

I understand that this is a personal tutorial project, but I'd like to kindly suggest that if you have the time, you can implement this so your code follows the requirements.